src/Components/v2/ArtworkFilter/ArtworkFilterArtworkGrid2.tsx

```diff
@@ -0,0 +1,158 @@
+import { Box, Spacer } from "@artsy/palette"
+import { ArtworkFilterArtworkGrid2_filtered_artworks } from "__generated__/ArtworkFilterArtworkGrid2_filtered_artworks.graphql"
+import { AnalyticsSchema, useSystemContext, useTracking } from "Artsy"
+import ArtworkGrid from "Components/ArtworkGrid"
+import React, { useEffect, useState } from "react"
+import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
+import { LoadingArea } from "../LoadingArea"
+import { PaginationFragmentContainer as Pagination } from "../Pagination"
+import { useFilterContext } from "./ArtworkFilterContext"
+import { ArtworkFilterZeroState } from "./ArtworkFilterZeroState"
+
+interface ArtworkFilterArtworkGridProps {
+  columnCount: number[]
+  filtered_artworks: ArtworkFilterArtworkGrid2_filtered_artworks
+  isLoading?: boolean
+  relay: RelayRefetchProp
+  term: string
```

Comment:
`term` is specific to the Search app's usage of the artwork grid. We'll need to figure out a way to opt into this prop only for the apps that need it. 

Maybe that's just marking it as optional, and letting the apps decide what they pass in. When we get to the artist app, that one would pass in an optional `artist` or `artist_id`, too.